### ISSUE TITLE
[3.5.0] bottom perimeter / layers shows the inner part (request?)

### ISSUE THREAD

**Issue by Liger0** [Ultimaker/Cura#4761]
**Application Version**
[3.5.0]

**Platform**
Win 10 x64

**Printer**
Cr-10S

**Steps to Reproduce**
Use a model which has a complex shape on the bottom

**Actual Results**
the infill is shown from the bottom of the model because there isn't any perimeter

**Expected results**
The perimeter or bottom layer should follow its shape

**Additional Information**
![infill from the bottom](https://user-images.githubusercontent.com/20385116/48225416-0e0e7800-e39d-11e8-86ff-ddc1427a48a5.png)
[CCR10S_45 ACP.zip](https://github.com/Ultimaker/Cura/files/2568495/CCR10S_45.ACP.zip)

**Comment by diegopradogesto** [comment]
Please attach the project file instead of only the STL, so that we can see exactly all your settings.

**Comment by Liger0** [comment]
> 
> 
> Please attach the project file instead of only the STL, so that we can see exactly all your settings.

I updated with the .3mf file. Thank you.

**Comment by diegopradogesto** [comment]
I was hardly investigating this and I know where the problem is. The culprit is the "Skin Removal Width" that depends on the "Inner Wall(s) Line Width". In this case the line width was set to 0.45mm (instead of the default 0.4mm), that makes the removal width to be 1.35mm. With this value and no expansion afterwards (setting "Skin Expand Distance" to 0) all the skin on top of some of the letters disappears completely:
![image](https://user-images.githubusercontent.com/15830521/48311567-dc1a3300-e5a1-11e8-81d0-7a82c50f4644.png)

Then the expand distance, even if it's set to 1.35mm, there is no skin to expand in these parts, and so the skin is not generated there.

In summary, there is nothing wrong with the behavior, it's only a matter of some settings' value, that for this specific model makes this lack of skin. You can try different things for this model:

- Use "Inner Wall(s) Line Width" = 0.4mm.
- Reduce the value of "Skin Removal Width" and "Skin Expand Distance" to 1.25mm.
